src/lib/helpers.ts

```diff
@@ -32,10 +32,19 @@ export async function initClients(provider: JsonRpcProvider, searcherPublicKey:
   // Create an Ethereum wallet signer using the derived private key, connected to the provided JSON RPC provider.
   const authSigner = new Wallet(derivedPrivateKey).connect(provider);
 
+  // Use custom network for MevShare and connect for FlashbotsBundle as we might need adding x-flashbots-origin headers.
+  const network = {
+    streamUrl: SupportedNetworks.mainnet.streamUrl,
+    apiUrl: env.forwardUrl,
+    apiHeaders: env.flashbotsOrigin !== "" ? { "x-flashbots-origin": env.flashbotsOrigin } : undefined,
+  };
+  const connect = new ethers.FetchRequest(env.forwardUrl);
```

Comment:
I've never seen this construction before -- it's compatible with FB bundle provider and ethers as a `ConnectionInfo` struct?

Reply:
this is specific to the forked FB bundle client that is using ethers v6, and is expecting FetchRequest here